webapp/package.json

```
@@ -28,8 +28,9 @@
     "@angular/platform-browser-dynamic": "18.2.1",
     "@angular/router": "18.2.1",
     "@devolutions/icons": "4.0.8",
-    "@devolutions/iron-remote-gui": "^0.11.6",
-    "@devolutions/iron-remote-gui-vnc": "^0.2.2",
+    "@devolutions/iron-remote-desktop": "0.2.0",
+    "@devolutions/iron-remote-desktop-rdp": "0.1.1",
+    "@devolutions/iron-remote-desktop-vnc": "0.2.0",
```

suggestion: As long as we follow the semantic versioning, it’s fine to use `^` .
```suggestion
    "@devolutions/iron-remote-desktop": "^0.2.0",
    "@devolutions/iron-remote-desktop-rdp": "^0.1.1",
    "@devolutions/iron-remote-desktop-vnc": "^0.2.0",
```

---

Done